docs/HISTORY.txt

```diff
@@ -2,10 +2,11 @@ Changelog
 =========
 
 
-1.3.1 (unreleased)
+1.3.1 (2018-09-11)
```

Comment:
This will be set automatically when we actually release that version (we release our packages with [`zest.releaser`](https://pypi.org/project/zest.releaser/), and it will automatically enter the date of the release day when doing so). So could you please undo this and change it back to `unreleased`, otherwise `zest.releaser` will get confused 😜